### DISCUSSION TITLE
What's new page still doesn't show the latest version on desktop

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#15091] [Q&A]
This seems to still be the case: https://github.com/ravendb/ravendb/discussions/13583

When I open the whats-new page for 5.3 (https://ravendb.net/docs/article-page/5.3/csharp/start/whats-new) on a mobile phone, the page shows the latest version (5.3.106). However, when I open it on my laptop, the last version shown is 5.3.105. This also happens with a direct request using curl: [whats-new.zip](https://github.com/ravendb/ravendb/files/9747599/whats-new.zip)

The command used was:

`curl -i -v https://ravendb.net/docs/article-page/5.3/csharp/start/whats-new > whats-new.html 2>&1`

The devices share the internet connection.

**Comment by arekpalinski** [comment]
For that it happened again. Fixed.
